package.json

```diff
@@ -174,10 +174,14 @@
                 },
                 "rust-client.channel": {
                     "type": [
-                        "string",
-                        "null"
+                        "string"
                     ],
-                    "default": null,
```

Comment:
I think changing the string to a set of options is a great thing to do, however, I think the default should still be `null` since in that case we'll use whichever channel the user would normally use and that is probably the expected behaviour (e.g., using nightly if the user would usually build with nightly).

Reply:
Ahh, I didn't know that setting the value to ``null`` would do that! However, I do not think that ``null`` communicates "RLS updates itself based on the settings of your current toolchain config", can we maybe improve that somehow? 

I for instance thought that it would default to stable (and have thus overridden the channel for every project that I have)

Reply:
We could add some documentation of that to the tool tip?

Reply:
I've gone ahead and added some explanation to the description. If you want it somewhere else feel free to say so :)